src/main/antlr4/imports/mysql_alter_table.g4

```diff
@@ -65,3 +65,9 @@ ignored_alter_specifications:
     ;
   algorithm_type: DEFAULT | INPLACE | COPY;
   lock_type: DEFAULT | NONE | SHARED | EXCLUSIVE;
+
+alter_ordering: alter_ordering_column (ASC|DESC)?;
+alter_ordering_column:
+    name '.' name '.' name
```

Comment:
would `table_name '.' name | name;` work here? does it matter adding a test for that case?